packages/walt-compiler/src/__tests__/compiler-spec.js

```diff
@@ -109,13 +109,17 @@ test("compiles math", t =>
 test("invalid imports throw", t =>
   t.throws(() => compile("import foo from 'bar'")));
 
-test.skip("64 bit constant encoding", t => {
+test("64 bit constant encoding", t => {
   return compileAndRun(`
     function number(): i64 {
-      return 42;
+      const x: i64 = 42;
+      return x;
+    }
+    function two() : i64 {
+      return 2;
     }
     export function test(): i32 {
       return number(): i32;
     }
-  `).then(result => t.is(result.instance.exports.test(), 0));
+  `).then(result => t.is(result.instance.exports.test(), 42));
```

Comment:
Just for my information, the `test` function is doing a conversion from `i64` to `i32`? Is this your technique to test 64 bits ints in js?

Reply:
Yes, but 64-bit nums are tested indirectly. If that encoding failed, then the data would be garbage or the program wouldn't compile at runtime.

The `:` is a [type cast operator borrowed from flow](https://flow.org/en/docs/types/casting/#toc-type-cast-expression-syntax)